sphinxcontrib/confluencebuilder/config/exceptions.py

```diff
@@ -62,6 +62,17 @@ def __init__(self, msg):
 
 The option 'confluence_default_table_width' has been provided to override the
 default width for tables in editor v2. Accepted values include a string or a positive integer.
+If the value is a string one can define a unit otherwise 'px' is internally used as default.
+Accepted units include:
+
+ - px
+ - em
+ - ex
+ - mm
+ - cm
+ - in
+ - pt
+ - pc
```

Comment:
I do not believe it will be necessary to explicitly list unit sizes here. We aim to provide support for various unit sizes defined by the markup used (e.g. [rst](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#length-units)). Could write something like this:

> String values will be interpreted by units supported by the markup processed. Strings without units or integer values will be interpreted as a pixel value.

And we do not have to maintain this list as we support new unit types over time.